pkg/proc/exec.go

```diff
@@ -0,0 +1,70 @@
+package proc
```

Comment:
Ugg, I moved these files to `pkg/proc/` instead of `pkg/client/cli/cliutil/` and the diff viewer can't figure out that they're the same file to show a useful diff.